buildSrc/src/main/kotlin/Project.kt

```diff
@@ -97,7 +97,7 @@ object Dependencies {
   const val composeUi = "androidx.compose.ui:ui:${Versions.compose}"
   const val composeMaterial = "androidx.compose.material:material:${Versions.compose}"
   const val composeUiToolingPreview = "androidx.compose.ui:ui-tooling-preview:${Versions.compose}"
-
+  const val composeUiTooling = "androidx.compose.ui:ui-tooling:${Versions.compose}"
```

Comment:
probably better to split compose dependencies to another object inside dependencies. so we can access them like `Dependencies.Componse.**`

Reply:
Ideally we will switch to [Bill Of Materials](https://developer.android.com/jetpack/compose/bom/bom) and version catelogs after we bump Kotlin version and AGP version later. For now let's keep it simple :)